pom.xml

```diff
@@ -3,9 +3,13 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.google.cloud.tools</groupId>
-  <artifactId>appengine-maven-plugin</artifactId>
+  <artifactId>appengine-maven</artifactId>
   <version>1.2.2-SNAPSHOT</version>
-  <packaging>maven-plugin</packaging>
+  <modules>
+    <module>appengine-maven-parent</module>
+    <module>appengine-maven-plugin</module>
+  </modules>
+  <packaging>pom</packaging>
```

Comment:
Since this is a pom now, I would move the whole `<dependencies>` block in this parent to `appngine-maven-plugin\pom.xml`.